unlock-app/src/components/interface/locks/Settings/forms/SubscriptionForm.tsx

```diff
@@ -71,17 +84,67 @@ export const SubscriptionForm = ({
   }
 
   const RecurringDescription = () => {
-    if (recurringPossible) return null
-    return (
-      <small className="text-sm text-brand-dark">
-        Recurring memberships are only available for locks that are using and
-        ERC20 currency and that have expirations.
-      </small>
-    )
+    const termsSettingsUrl = `/locks/settings?address=${lockAddress}&network=${network}&defaultTab=terms`
+
+    if (isLoading) return null
+    if (isRecurringPossible) return null
+
+    if (lock?.publicLockVersion >= 10) {
+      return (
+        <div className="grid  gap-1.5">
+          <small className="text-sm text-brand-dark">
+            Recurring memberships are only available for locks that are using
+            and ERC20 currency and that have expirations and a refund value.
+          </small>
+          <ul className="ml-2 list-disc">
+            {refund === 0 && (
```

Comment:
We should have a specific gasRefund value that we use for each network.

Reply:
is actually as-in, refund value comes from `getIsRecurringPossible` and is for every `lockAddress + network ` combination